src/table/expandable-rows/expand-toggle-button.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useRef } from 'react';
+import { useSingleTabStopNavigation } from '../../internal/context/single-tab-stop-navigation-context';
+import styles from './styles.css.js';
+import InternalIcon from '../../icon/internal';
+import { ExpandableItemProps } from './expandable-rows-utils';
+
+export function ExpandToggle({ isExpandable, ...props }: ExpandableItemProps) {
+  return isExpandable ? <ExpandToggleButton {...props} /> : <ExpandTogglePlaceholder />;
+}
+
+function ExpandToggleButton({
+  isExpanded,
+  onExpandableItemToggle,
+  expandButtonLabel,
+  collapseButtonLabel,
+}: Omit<ExpandableItemProps, 'isExpandable'>) {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const { tabIndex } = useSingleTabStopNavigation(buttonRef);
+  return (
+    <button
+      ref={buttonRef}
+      tabIndex={tabIndex}
+      aria-label={isExpanded ? collapseButtonLabel : expandButtonLabel}
+      aria-expanded={isExpanded}
+      className={styles['expand-toggle']}
+      onClick={onExpandableItemToggle}
+    >
+      {isExpanded ? (
```

Comment:
In expandable section we turn the icon 90 degrees with a small animation, instead of replacing it: https://github.com/cloudscape-design/components/blob/main/src/expandable-section/motion.scss#L18

Do we want to do the same here?

Reply:
@katiegeorge, what do you think?

Reply:
I added animation for the expand toggle. No animation for the content though.